src/components/tl/mlx5/tl_mlx5_team.c

```diff
@@ -20,12 +54,33 @@ UCC_CLASS_INIT_FUNC(ucc_tl_mlx5_team_t, ucc_base_context_t *tl_context,
 
     self->a2a    = NULL;
     self->dm_ptr = NULL;
+
+    if (ucc_topo_get_sbgp(UCC_TL_CORE_TEAM(self)->topo, UCC_SBGP_NODE)
+            ->group_rank == 0) {
+        status = ucc_tl_mlx5_dm_init(self);
```

Comment:
done. We gather the global status in `ucc_tl_mlx5_team_create_test` with a service allreduce.

@Sergei-Lebedev Question: in case team creation fails do I need to handle the tl_team cleanup manually or does core takes care of calling the cleanup function ?